keyboards/boardsource/3x4/rules.mk

```
@@ -29,3 +29,4 @@ UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+LAYOUT = 3x4
```

This should be:
```suggestion
LAYOUT = ortho_3x4
```

But changing this means that you need to change/add `LAYOUT_ortho_3x4`, as well.

Otherwise: 
```suggestion
```